docs/quickstart.md

```diff
@@ -1,179 +1,223 @@
+For all other domains, you can use one of the following options.
 
- * `storage_type` : Storage provisioner to configure. Supported values: nfs (For now only nfs provisioner is supported, any other value won't setup a storageclass)
- * `volume_size` : If storage_type is nfs, a volume will be created with given size in GB and attached to bastion node. Eg: 1000 for 1TB disk.
+1. **Add entries to your DNS server**
 
-### Setup OCP Upgrade Variables
+    The general format is shown below:
+    ```
+    api.<cluster_id>.  IN  A  <bastion_public_ip>
+    *.apps.<cluster_id>.  IN  A  <bastion_public_ip>
+    ```
+    You'll need `bastion_public_ip` and `cluster_id`. This is printed at the end of a successful install. Or you can retrieve it anytime by running `terraform output` from the install directory.
+    For example, if `bastion_public_ip = 16.20.34.5` and `cluster_id = test-cluster-9a4f` then the following DNS records will need to be added.
+    ```
+    api.test-cluster-9a4f.  IN  A  16.20.34.5
+    *.apps.test-cluster-9a4f.  IN  A  16.20.34.5
+    ```
 
-Update the following variables specific to OCP upgrade. The upgrade will be performed after a successful install of OCP.
+2. **Add entries to your client system `hosts` file**
 
- * `upgrade_version` : (Optional) OpenShift higher and supported version. If set, OCP cluster will be upgraded to this version. (e.g. `"4.5.4"`)
- * `upgrade_channel` : (Optional) OpenShift channel having required upgrade version available for cluster upgrade. By default it is automatically set to stable channel of installed cluster (eg: stable-4.5). See [Understanding Upgrade Channels](https://docs.openshift.com/container-platform/4.5/updating/updating-cluster-between-minor.html#understanding-upgrade-channels_updating-cluster-between-minor) for more information on setting the upgrade channel.
- * `upgrade_pause_time` : (Optional) Minutes to pause the playbook execution before starting to check the upgrade status once the upgrade command is executed.
- * `upgrade_delay_time` : (Optional) Seconds to wait before re-checking the upgrade status once the playbook execution resumes.
+    For Linux and Mac `hosts` file is located at `/etc/hosts` and for Windows it's located at `c:\Windows\System32\Drivers\etc\hosts`.
 
-## Setup Data Files
-You need to have following files in data/ directory before running the Terraform templates.
-```
-$ ls data/
-id_rsa  id_rsa.pub  pull-secret.txt
-```
- * `id_rsa` & `id_rsa.pub` : The key pair used for accessing the hosts. These files are not required if you provide `public_key_file` and `private_key_file`.
- * `pull-secret.txt` : File containing keys required to pull images on the cluster. You can download it from RH portal after login https://cloud.redhat.com/openshift/install/pull-secret.
+    The general format is shown below:
+    ```
+    <bastion_public_ip> api.<cluster_id>
```

Comment:
Better keep it `bastion_ip` instead of `bastion_public_ip` everywhere to avoid confusion.